roles/dd-nodepool/files/nodepool_server.py

```diff
@@ -55,3 +70,25 @@ def get_nodepool_status(self, status_url, timeout, service_check_tags):
                            AgentCheck.OK,
                            message='Received status from %s' % status_url,
                            tags=service_check_tags)
+
+    def get_nodepool_images(self, images_url, timeout, service_check_tags):
+        r = requests.get(images_url, timeout=timeout)
+        currentTime = int(time.time())
+        staleImage = False
+
+        for image in r:
+            if currentTime - image.age > 172800:
```

Comment:
In either case, this'll be an attribute error.  If updated to do the json thing, you can access dict items instead:

``
if currentTime - image['age'] > 172800:
``